src/hb-common.h

```diff
@@ -545,6 +546,7 @@ typedef enum
   HB_SCRIPT_HANUNOO			= HB_TAG ('H','a','n','o'), /*3.2*/
   HB_SCRIPT_TAGALOG			= HB_TAG ('T','g','l','g'), /*3.2*/
   HB_SCRIPT_TAGBANWA			= HB_TAG ('T','a','g','b'), /*3.2*/
+  HB_SCRIPT_MATH			= HB_TAG ('Z','m','t','h'), /*3.2*/
```

Comment:
These constants are grouped by which HarfBuzz version they were released in, so `HB_SCRIPT_MATH` should be added in a new section, right before `HB_SCRIPT_INVALID`, with a “Since” comment like the other sections have.

Reply:
They're grouped by the Unicode version they are added in. In this case, let's group it before UNKNOWN, with an empty Unicode version number.

Reply:
Ignore my comment. What David said.